config/conversion_webhook/secret.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: webhook-certificate
+data:
+  tls.crt: ""
```

Comment:
Won't this be a problem in case of managed installation with KLM?

Reply:
Since Kyma Lifecycle Manager uses the manifest yaml to install the module, the secret will most likely be reverted

Reply:
Ah you mean on new version, each apply of the manifest will clear out the secret and we regenerate on each upgrade. This is not optimal, let me think of a solution to this.

Reply:
Implemented workaround solution without mounting the secret. Digging more into KLM and their concepts won't bring much.

Reply:
Thanks, current approach looks good. Even though it might be a workaround, I like the approach where we don't mount the secret directly to the operator, as it will avoid any issue with stored value of the cert being different then the Secret value.